config/default.yml

```diff
@@ -2792,6 +2860,18 @@ Style/HashSyntax:
   # Do not suggest { a?: 1 } over { :a? => 1 } in ruby19 style
   PreferHashRocketsForNonAlnumEndingSymbols: false
 
+Style/HashTransformKeys:
+  Description: 'Prefer `transform_keys` over `each_with_object` and `map`.'
+  Enabled: true
```

Comment:
That's cool, I did not know about this. :+1: 

However, I fear this might introduce issues with existing projects:

> This cop should only be enabled on Ruby version 2.5 or newer (transform_keys was added in Ruby 2.5.)

(<https://github.com/rubocop-hq/rubocop/blob/v0.81.0/manual/cops_style.md#stylehashtransformkeys>)

So maybe disable? Or just keep it, as I can't remember when we actually only transformed keys of a Hash.